packages/vite/src/node/plugins/dataUri.ts

```diff
@@ -39,7 +39,7 @@ export function dataURIPlugin(): Plugin {
       const [, mime, format, data] = match
       if (mime !== 'text/javascript') {
         throw new Error(
-          `data URI with non-JavaScript mime type is not supported. If you're using legacy JavaScript MIME types (such as 'application/javascript'), please use 'text/javascript' instead.`,
+          `data URI with non-JavaScript mime type is not supported. If you're using legacy JavaScript MIME types (such as 'text/javascript'), please use 'text/javascript' instead.`,
```

Comment:
```suggestion
          `data URI with non-JavaScript mime type is not supported. If you're using legacy JavaScript MIME types (such as 'application/javascript'), please use 'text/javascript' instead.`,
```
I think this should be as-is, otherwise the sentence doesn't make sense.

Reply:
Good catch!